### DISCUSSION TITLE
Choose whether the menu should have a style to control the position of the content, such as left and right.

### DISCUSSION THREAD

**Discussion by ZhangChengLin** [twbs/bootstrap#30875] [General] [answered]
https://getbootstrap.com/docs/4.5/components/forms/#select-menu

1.Choose whether the menu should have a style to control the position of the content, such as left and right.

https://twbs-bootstrap.netlify.app/

2. Is the content of this website updated in real time? Apart from the version number v5.0.0-alpha1, is there a more detailed display? For example, update date.

**Comment by mdo** [comment]
Select menus don't have this option in HTML, so if you want that, you need to use a dropdown.

And that v5 site is updated as things are merged into `master` I believe, so it's nearly up to date but not an official v5 alpha.

**Comment by ZhangChengLin** [comment]
https://getbootstrap.com/docs/4.5/components/forms/#select-menu

```
<select class="custom-select" dir='rtl'>
  <option selected>Open this select menu</option>
  <option value="1">One</option>
  <option value="2">Two</option>
  <option value="3">Three</option>
</select>
```

Whether the left and right control of the text position can be added is not an arrangement method.

**Comment by mdo** [comment, marked as answer]
To reiterate, `<select>` menus cannot have their position controlled. If you want something like a select menu with control over where your menus appears, you need to use our dropdown component: https://getbootstrap.com/docs/4.5/components/dropdowns/#examples.

**Comment by ZhangChengLin** [comment]
select {
    direction: rtl;
}
select option {
    direction: ltr;
}
